api/apiclient_test.go

```diff
@@ -94,7 +95,7 @@ func (s *apiclientSuite) TestConnectWebsocketMultipleError(c *gc.C) {
 	info := s.APIInfo(c)
 	addr := listener.Addr().String()
 	info.Addrs = []string{addr, addr, addr}
-	_, _, err = api.ConnectWebsocket(info, api.DialOpts{})
+	_, _, err = api.DialAPI(info, api.DialOpts{})
 	c.Assert(err, gc.ErrorMatches, `unable to connect to API: websocket.Dial wss://.*/model/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}/api: .*`)
```

Comment:
For the record, I'm not requesting that you fix every test, just modify the error message in the test you're fixing. You mentioned wanting to check the entire error message, so I suggested a new test to check just that.

Pursuant to your previous comment about not wanting test-fixes in this PR, please open another PR with the test modified and ping me for review.